tests/unit/models/test_food_analysis.py

```
@@ -0,0 +1,172 @@
+class TestFoodAnalysisRequest:
+    """Tests for the FoodAnalysisRequest model."""
+    
+    def test_food_analysis_request_creation(self):
+        """Test creating a food analysis request."""
+        request = FoodAnalysisRequest(description="A grilled chicken sandwich")
+        assert request.description == "A grilled chicken sandwich"
+    
+    def test_food_analysis_request_validation(self):
+        """Test request validation."""
+        # Missing description should raise error
+        with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)